vispy/visuals/image.py

```diff
@@ -39,8 +39,8 @@
     // Cast ray from 3D viewport to surface of image
     // (if $transform does not affect z values, then this
     // can be optimized as simply $transform.map(x) )
-    vec4 p1 = $transform(x);
-    vec4 p2 = $transform(x + vec4(0, 0, 0.5, 0));
+    vec4 p1 = $polar_transform($transform(x));
+    vec4 p2 = (p1 + vec4(0, 0, 0.5, 0));
```

Comment:
Not sure if this is correct. Originally the `z=0.5` offset was applied in visual/data coordinates, now it's applied after the model and polar transform.

Reply:
Well spotted. I already wondered why the polar image is above the other visual in any case.

Reply:
Done!